app/views/api/asciicasts/show.html.slim

```diff
@@ -19,6 +19,14 @@ javascript:
       if (e.data == 'asciicast:play' || e.data[0] == 'asciicast:play') {
         player.play();
       }
+
+      // reveal.js support
+
+      if (e.data == 'slide:start') {
```

Comment:
We already make use of `data-autoplay`. This is example from [embedding doc](https://asciinema.org/docs/embedding):

    <script type="text/javascript" src="https://asciinema.org/a/14.js" id="asciicast-14" async data-autoplay="true" data-size="big"></script>

It starts the playback automatically on player initialization. So if you use the above script in your slides it will start playing immediately when the presentation is open.

What just came to my mind is, we could extend `data-autoplay` to accept either `true` or `revealjs` as a value. The former would behave as it is now, the latter would integrate `slide:start/stop` events instead of immediately starting. What do you think?